### PR TITLE
Connorwon 237 update backend security frontend changes

### PR DESCRIPTION
**Pull request by ConnorWon**
## Description

Refactoring frontend based on changes to backend security

## Updates

Updated all axios requests to no longer send access token in authorization header
Axios requests now use withCredentials setting to send access & refresh token cookies
Made changes to auth redux to work with new implementation of access & refresh token

## Checklist

- [x] Ran `npm run lint:fix` to format code
- [x] Requested at least one reviewer
- [x] Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

**Comment by ConnorWon** [comment]
[Update Security for Backend](https://trello.com/c/RcrTUCq1/237-update-security-for-backend)

**Comment by ElisaHu1** [comment]
Error: Authentication error on our end.
I am not able to log in with my credential anymore, if that is intended, can you give me the new credential

**Review by ElisaHu1 — APPROVED** [review]

File: src/pages/Timeline/EditTimelineEntry/EditTimelineEntry.tsx [hunk 1]
```diff
@@ -64,13 +61,11 @@ const EditTimelineEntry = () => {
 
   const save = async () => {
     const updatedTimeline = {...timeline, contributors: timeline.contributors.split(",").map((c: string) => c.trim())};
+    console.log(updatedTimeline);
```
Comment (ryanyae) [inline src/pages/Timeline/EditTimelineEntry/EditTimelineEntry.tsx:64]: Do you mean to keep this?

**Review by ryanyae — APPROVED** [review]
All looks good to me, good job

File: src/pages/Timeline/EditTimelineEntry/EditTimelineEntry.tsx (hunk 1 above)
Comment (ConnorWon) [reply src/pages/Timeline/EditTimelineEntry/EditTimelineEntry.tsx:64]: Oh yeah we can get rid of that

**Pull request merged**